### PR TITLE
Added support of Service Accounts management

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

File: neptune/management/internal/api.py [hunk 1]
```diff
@@ -433,3 +437,140 @@ def get_workspace_member_list(name: str, api_token: Optional[str] = None) -> Dic
         }
     except HTTPNotFound as e:
         raise WorkspaceNotFound(workspace=name) from e
+
+
+@with_api_exceptions_handler
+def _get_raw_workspace_service_account_list(
+    name: str, api_token: Optional[str] = None
```
Comment (shnela) [inline neptune/management/internal/api.py:444]: Can we keep workspace_name and project_name variables in this file?

File: neptune/management/internal/api.py [hunk 2]
```diff
@@ -433,3 +437,140 @@ def get_workspace_member_list(name: str, api_token: Optional[str] = None) -> Dic
...
+    verify_type("api_token", api_token, (str, type(None)))
+
+    backend_client = _get_backend_client(api_token=api_token)
+    project_identifier = normalize_project_name(name=name, workspace=workspace)
+
+    params = {"projectIdentifier": project_identifier, **DEFAULT_REQUEST_KWARGS}
+
+    try:
+        result = backend_client.api.listProjectServiceAccounts(**params).response().result
+        return {
```
Comment (shnela) [inline neptune/management/internal/api.py:487]: You return dict from `get_project_service_account_list` function. Change one of that :p

**Review by shnela — CHANGES_REQUESTED** [review]

File: neptune/management/internal/api.py (hunk 1 above)
Comment (Raalsky) [reply neptune/management/internal/api.py:444]: Except for this function, the argument names are a part of product specification :wink:. Leave a comment there if you would like to discuss anything.

File: neptune/management/internal/api.py (hunk 2 above)
Comment (Raalsky) [reply neptune/management/internal/api.py:487]: Huh? This is `get_project_service_account_list`

**Review by shnela — APPROVED** [review]

**Review by shnela — APPROVED** [review]

**Pull request merged**